fiat-roles/src/test/groovy/com/netflix/spinnaker/fiat/redis/EmbeddedRedis.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.redis;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.embedded.RedisServer;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class EmbeddedRedis {
+
+  private final URI connection;
+  private final RedisServer redisServer;
+
+  private JedisPoolSource poolSource;
+
+  private EmbeddedRedis(int port) throws IOException, URISyntaxException {
```

Comment:
any particular reason for this as opposed to a dependency on [kork-jedis-test](https://github.com/spinnaker/kork/tree/master/kork-jedis-test) ?